splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/RadioComponent.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import RadioBar from '@splunk/react-ui/RadioBar';
+
+class RadioComponent extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    handleChange = (e) => {
+        this.props.handleChange(this.props.id, e.target.value)
+    };
+
+    render() {
+        return (
+            <RadioBar
+                onChange={this.handleChange}
+                value={this.props.value}
+                key={this.props.field}
+                style={{ width: 200 }}
```

Comment:
just wondering, Do we explicitly need to set width here? because what if our label has long name ? If you've check with diff. scenarios then it's fine.

Reply:
Updated and used **inline** argument which is working as expected with different scenarios.